sinatra/lib/tasks/docker_lint.rake

```
@@ -0,0 +1,21 @@
+        puts "#{dockerfile} not found in the project root."
+        exit 1
+      end
+  
+      # Run Hadolint using the local executable
+      puts "Running Hadolint on #{dockerfile}..."
+      unless system("./hadolint.exe #{dockerfile}")
+        puts "Hadolint found issues!"
+        exit 1
+      else
+        puts "No issues found. Good job!"
+      end
```

RuboCop (1.73)

[convention] Do not use `unless` with `else`. Rewrite these with the positive case first.

(Style/UnlessElse)